index.js

```diff
@@ -41,32 +41,47 @@ app.get('/',
         }
 
         if (typeof greet != 'undefined' && greet === "true") {
-            //let randomMessage = hours < 12 ? uniqueRandomArray(morningGreetings.greetings) : hours < 18 ? uniqueRandomArray(dayGreetings.greetings) : hours < 21 ? uniqueRandomArray(eveningGreetings.greetings) : uniqueRandomArray(nightGreetings.greetings);
             let randomMessage;
-            if(hours < 12){
+            if (hours < 12) {
                 randomMessage = uniqueRandomArray(morningGreetings.greetings)
-            }else if(hours >= 12 && hours < 18)
-            {
+            } else if (hours >= 12 && hours < 18) {
                 randomMessage = uniqueRandomArray(dayGreetings.greetings)
-            }else if(hours >=18 && hours < 21){
+            } else if (hours >= 18 && hours < 21) {
                 randomMessage = uniqueRandomArray(eveningGreetings.greetings)
-            }else{
+            } else {
                 randomMessage = uniqueRandomArray(nightGreetings.greetings)
             }
-            
+
             message = randomMessage();
             textMessage = randomMessage();
         }
-        const width = 1200
-        const height = 250
 
-        registerFont("poppins-Regular.ttf", {family: "Poppins", weight: "400", style: "Regular"})
-        registerFont("poppins-Bold.ttf", {family: "Poppins", weight: "700", style: "Bold"})
+        let width = 0
+        let height = 0
+
+        if (typeof w != 'undefined' && w !== '') {
+            width = w;
+        }
+
+        if (typeof h != 'undefined' && h !== '') {
+            height = h;
+        }
+
+        registerFont("poppins-Regular.ttf", { family: "Poppins", weight: "400", style: "Regular" })
+        registerFont("poppins-Bold.ttf", { family: "Poppins", weight: "700", style: "Bold" })
         const canvas = createCanvas(width, height)
         const context = canvas.getContext('2d')
-        const [foreground, background] = getColorPair();
 
+        const colors = uniqueRandomArray(colorPairs.colorPairs);
+        let background = `${colors()[0]}`;
+        let foreground = `${colors()[1]}`;
```

Comment:
use object  and properties instead of array. something like foreground and background
{
'background' : "#888",
'foreground' : "#FFF"
}

 and also update the colors.js